### PR TITLE
Skipping Travis/Appveyor build for non-code changes

### PR DESCRIPTION
**Pull request by matrix**
Hi,
I added rules in Travis/Appveyor to not run tests if no code changes are detected (must be tested).

### CODE REVIEW

**Comment by matrix** [comment]
First auto-test seems ok :)

File: .appveyor.yml [hunk 1]
```diff
@@ -56,3 +56,10 @@ test_script:
       }
 
       & $env:BASH -lc "cd '$env:APPVEYOR_BUILD_FOLDER' && ./hashcat.exe -m 0 --show *ple0.hash"
+
+only_commits:
+  files:
+    - src/**/*
+    - include/*
+    - deps/**/*
+    - Makefile
```
Comment (jsteube) [inline .appveyor.yml:66]: Please add OpenCL/inc_* files. Some modules make direct use of them.

**Comment by jsteube** [comment]
Great Idea

File: .appveyor.yml (hunk 1 above)
Comment (matrix) [reply .appveyor.yml:66]: ok, so I need update also the travis script to don't skip build if OpenCL/inc_* files are changed

**Pull request merged**